include/vfn/vfio/container.h

```diff
@@ -94,13 +94,11 @@ int vfio_map_vaddr_ephemeral(struct vfio_container *vfio, void *vaddr, size_t le
  */
 int vfio_unmap_ephemeral_iova(struct vfio_container *vfio, size_t len, uint64_t iova);
 
-#ifndef VFIO_IOMMU_TYPE1_INFO_CAP_IOVA_RANGE
-struct vfio_iova_range {
-	__u64	start;
-	__u64	end;
+struct libvfn_iova_range {
```

Comment:
I'd prefer to name this either `iommu_iova_range` or just `iova_range` if possible. *If* we should prefix it, it should be prefixed with just `vfn_`.

Reply:
went with iova_range